packages/wallets/infinitywallet/README.md

```diff
@@ -0,0 +1,5 @@
+# `@solana/wallet-adapter-infinitywallet`
```

Comment:
Please change this and other package name references to `@solana/wallet-adapter-infinity` and the directory to `infinity`.

Reply:
This was marked as resolved but hasn't changed.

Reply:
It won't be changed as we are not Infinity, our brand is Infinity Wallet as clearly explained already!